confluence-xml/src/main/java/org/xwiki/contrib/confluence/filter/input/ConfluenceXMLPackage.java

```diff
@@ -1816,6 +1816,11 @@ private String fixCDataAndNL(String elementText)
             .replace('\u2029', '\n');
     }
 
+    private String escapeWindowsFolderName(String folderName)
+    {
+        return folderName.replace(":", "=");
```

Comment:
Isn't there a risk of having an id with `=` ?

Better be safe than sorry and introduce a safer escaping syntax.